docs/integrations/sources/cockroachdb.md

```diff
@@ -95,6 +95,7 @@ Your database user should now be ready for use with Airbyte.
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.1.4 | 2021-12-08 | [8494](https://github.com/airbytehq/airbyte/pull/8494) | Update fields in source-connectors specifications |
```

Comment:
Please, set this changelog below, to  **Changelog source-cockroachdb-strict-encrypt** and set version `0.1.1`